Assets/Live2D/Cubism/Rendering/CubismRenderer.cs

```diff
@@ -673,6 +673,7 @@ private void TryInitializeMesh()
                     uv = drawable.VertexUvs,
                     triangles = drawable.Indices
                 };
+                mesh.name = "Live2D";
```

Comment:
We want to use drawable name for mesh name.
So, if you want to use this name, please use on your own branch.

Reply:
Drawable name could work too. I just want to avoid empty ones. I'll fix it